app/views/addresses/index.html.erb

```diff
@@ -0,0 +1,38 @@
+<h1>Available Addresses </h1>
+<% if @address.present? %>
+  <table class = "table"> 
+    <thead>
+      <tr>
+        <th scope = "col">S.No.</th>
+        <th scope = "col">Country</th>
+        <th scope = "col">State</th>
+        <th scope = "col">City</th>
+        <th scope = "col">Address</th>
+        <th scope = "col">Pincode</th>
+        <th scope = "col">Options</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @address.each_with_index do |addr, index_no| %>
```

Comment:
I think you can specify start value of index

Reply:
when i am trying to use:
@address.each_with_index(1) it is showing following error:
wrong number of arguments (given 1, expected 0)
that's why i didn't specify the value of index